03-transactions.md

```diff
@@ -117,11 +117,51 @@ If a revoked commitment transaction is published, the other party can spend this
 
 #### `to_remote` Output
 
-This output sends funds to the other peer and thus is a simple P2WPKH to `remotepubkey`.
+This output sends funds to the other peer, thus is not encumbered by a
+revocation private key.
+
+If `option_anchor_outputs` applies to the commitment transaction, the `to_remote` output is delayed similarly to the `to_local` output, and is to a fixed key:
+
+    `max_to_self_delay`
+    OP_CSV
+    OP_DROP
+    <remote_pubkey>
+    OP_CHECKSIG
+
+The output is spent by a transaction with `nSequence` field set to `max_to_self_delay` (which can only be valid after that duration has passed) and witness:
+
+    <remote_sig>
+
+Otherwise, this output is a simple P2WPKH to `remotepubkey`.
+
+
+#### `to_local_anchor` and `to_remote_anchor` Output (option_anchor_outputs)
+
+This output can be spent by the local and remote nodes respectively to provide incentive to mine the transaction, using child-pays-for-parent. Both
+anchor outputs are always added, except for the case where there are no htlcs and one of the parties has a commitment output that is below the dust limit. 
+In that case only an anchor is added for the commitment output that does materialize. This typically happens if the initiator closes right after opening
+(no `to_remote` output).
+
+   OP_DEPTH
+   OP_IF
+       <funding_pubkey> OP_CHECKSIG
```

Comment:
Not clear _which_ funding pubkey this is.

Reply:
It is described in the sentence right below. But clarified it further by using `<local_funding_pubkey/remote_funding_pubkey>`